locale/pt_PT/editor.xml

```diff
@@ -37,7 +37,7 @@
 	<message key="editor.notifyUsers.interestedUsers">Enviar esta mensagem a todos os utilizadores associados a esta revista que indicaram desejar receber notificações ({$count} utilizadores)</message>
 	<message key="editor.notifyUsers.includeToc">Incluir sumário da edição:</message>
 	<message key="editor.editorAdministration">Administração do Editor</message>
-	<message key="editor.issues.createIssue">Criar Edição</message>
+	<message key="editor.issues.createIssue">Gerir Edições</message>
```

Comment:
Hi Marco, sorry, I didn't see this (and other) messages, don't know why. The problem with this translation is related to the context in which this is used because you can also "manage" the existing issues. 
As this is a direct link in the editor dashboard (User Home), it takes users to think about creating a new issue only, and is not clear how to "manage" or "edit" existing issues. 
Another option is to change the translation to "Create & Edit Issues"... 
As we are moving to a new interface, maybe is not an issue anymore ;-)